release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -208,6 +208,14 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 * An IPv6 network for API and Ingress services on the second `machineNetwork` configuration.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+[id="ocp-4-16-networking-metallb-frr-k8s_{context}"]
+==== Integration of MetalLB and FRR-K8s (Technology Preview)
+
+This release introduces `FRR-K8s`, a Kubernetes based DaemonSet that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
+As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure `MetalLB` to use `FRR-K8s` as the backend.
+You can use this to operate FRR services, such as receiving routes.
+
+For more information, see xref:../networking/metallb/metallb-frr-k8s.adoc[Configuring the integration of MetalLB and FRR-K8s].
```

Comment:
This[ xref needs an anchor id](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references). For detailed examples on how to format xrefs, see the [Cross-referencing by example](https://docs.google.com/document/d/1pavrM-4u8MEmuZFNF3wBeDmcmZi-kAZNRxp1Cze2CkQ/edit).

Reply:
Thanks for that. Updated accordingly.